.gitignore

```diff
@@ -1,6 +1,6 @@
 .idea
 .vscode
-venom
+cli/venom/bin
```

Comment:
```suggestion
cli/venom/bin
venom
```

Reply:
ok to ignore bin, but keep ignore venom to avoid push the bin venom produced with a simple go build in cli/venom directory